ulohy_10/class_currency.py

```diff
@@ -0,0 +1,45 @@
+import datetime
```

Comment:
Soubory mužou být pojmenovány normálně, prefix class je zbytečný.

Reply:
Já vím, to je asi tak nějak pro mě, abych věděla, že v tomto souboru je třída na ...